frontend/src/metabase-types/api/collection.ts

```diff
@@ -127,9 +128,28 @@ export type ListCollectionItemsRequest = {
   pinned_state?: "all" | "is_pinned" | "is_not_pinned";
   sort_column?: "name" | "last_edited_at" | "last_edited_by" | "model";
   sort_direction?: "asc" | "desc";
+  namespace?: "snippets";
 } & PaginationRequest;
 
 export type ListCollectionItemsResponse = {
   data: CollectionItem[];
   models: CollectionItemModel[] | null;
 } & PaginationResponse;
+
+export type CollectionRequest = {
+  id: CollectionId;
```

Comment:
`GET /api/collection/:id` cannot accept "personal" or "users" as defined in this type, so this will lead to errors if used incorrectly.

The correct type is `number | "root"`